libra/README.md

```diff
@@ -1,6 +1,7 @@
 ## Libra
+NOTE: This docker-compose (Apache only) is only needed for running libra on your local machine. The production app uses the orion Apache image (with the proxy).  
 
-In order to run this on your local machine you will need to start orion first using the docker-compose.dev.yml file to create the network and bring up the mongo db.
+In order to run this on your local machine, you will need to start orion first using the docker-compose.dev.yml file to create the network and bring up the Mongo DB and orion-data. Then shut down the orion apache (`docker stop apache`) so it doesn't interfere with the libra Apache. 
```

Comment:
Can we/should we set this up so that our local running version of this uses the apache instance in orion, rather than having a separate Apache instance for libra?  

Or, can we make a change to the dev docker-compose that contains a mongo store and contains the orion-data spring container so we don't have to mess around with shutting down the right Apache instance?

Reply:
Yes, we should write a card to figure out how to get these running at the same time. Libra was using/exposing a different port but I'm not sure how that was even working considering it was using a basic apache image. I would suggest a new card.